content/national-qualifications-framework/ncit/content/database-development/_index.md

```diff
@@ -82,8 +82,7 @@ ncit_standards:
 - 114048
 - 114049
 prerequisites:
-  hard:
-  - sql/shop-project
```

Comment:
are you sure? I know this is a prerequisite elsewhere but the thing about graphs is that they can have multiple links between nodes. It doesn't have to be a tree, it's more like a roadmap.